docs/user/ml/index.asciidoc

```diff
@@ -16,6 +16,8 @@ time field, you can use the *Data Visualizer* to identify possible fields for
 [role="screenshot"]
 image::user/ml/images/ml-data-visualizer-sample.jpg[Data Visualizer for sample flight data]
 
+WARNING: Data visualization from an uploaded file is an experimental feature.
+
 experimental[] You can also upload a CSV, NDJSON, or log file (up to 100 MB in 
```

Comment:
Doesn’t the `experimental[]` annotation here already achieve this?

Reply:
@droberts195 In theory, it does. However, according to #47620, the experimental flag on the UI doesn't raise attention and easy to miss when the reader skims the text. To avoid it, I thought it is better to use a WARNING flag to reinforce the statement.

Reply:
Let's see what @lcawl thinks, but I think it's a bit strange to have two different ways of flagging the feature as experimental adjacent to each other.  If we go with the more "in your face" way Kibana Maps 6.7 did it then we can surely remove the more discrete `experimental[]` tag that's right next to that.

But also, of all the experimental features that ML has I think this one is probably the least important to flag up.  This is because it's inherently hard to use as part of a production workflow, so the fact that it is likely to move to a different part of Kibana in the future is not likely to disrupt a production system.  The nature of the feature is that people will use it manually on their laptops rather than programmatically integrated in a way that makes changes to the feature painful to react to.

Reply:
Thank you @droberts195 for sharing your opinion. I close this PR as it is unnecessary.